test/integration/ValidatorTest.php

```diff
@@ -385,7 +381,7 @@ public function notEmptyProvider()
             'valid_int_zero' => [1, true],
             'valid_float'      => [1.5, true],
             'valid_float_zero' => [0.0, true],
-            'valid_object'     => [$validObject, true],
+            'valid_object'     => [$validObject, true], //todo check why it is failing
```

Comment:
I think it is redundant since the test it failing. I would remove this comment and fix it, cause according to the original implementation, we do not support length for objects.

```suggestion
            'valid_object'     => [$validObject, false],
```